files/__addonLocation__/babel.config.json

```diff
@@ -2,12 +2,10 @@
   "plugins": [
 <% if (typescript) { %>  ["@babel/plugin-transform-typescript", { "allExtensions": true, "onlyRemoveTypeImports": true, "allowDeclareFields": true }],
 <% } %>    "@embroider/addon-dev/template-colocation-plugin",
-    "@babel/plugin-transform-class-static-block",
     ["babel-plugin-ember-template-compilation", {
       "targetFormat": "hbs",
       "transforms": []
     }],
-    ["@babel/plugin-proposal-decorators", { "version": "legacy" }],
-    "@babel/plugin-proposal-class-properties"
+    ["decorator-transform", { "runtime": { "import": "decorator-transform/runtime" } }],
```

Comment:
Two things here: first, the package name is `decorator-transforms`! (plural)

But second, even when renaming this, looking at the CI failures, it seems babel expects a `babel-plugin-decorator-transform(s)`  package here, as it auto-prefixes it.

According to https://babeljs.io/docs/options#name-normalization, we should be able to use `module:decorator-transforms` here. Or `require.resolve` it, as shown in its docs. Or rename the package to `babel-plugin-decorator-transforms`? :) /cc @ef4

Reply:
Ye! I made this mistake again yesterday, atd ef4 corrected me, and pointed me at those docs.
🎉